example/src/Pages/BasicMap.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState, useRef } from 'react'
+import { LatLngExpression, Marker as MarkerType } from 'leaflet'
+import 'leaflet/dist/leaflet.css'
+import { Map, TileLayer, Marker } from '@datapunt/react-maps'
+import { ViewerContainer, SearchBar } from '@datapunt/asc-ui'
+import Controls from '../components/Zoom'
+import { constants } from '../../../src'
+import DefaultMarkerIcon from '../components/DefaultMarkerIcon'
+import NonTiledLayer from '../components/NonTiledLayer'
+
+const BasicMap = () => {
+  const [marker, setMarker] = useState<MarkerType>()
+  const [markerPosition, setMarkerPosition] = useState({
+    lat: 52.3731081,
+    lng: 4.8932945,
+  })
+
+  const [markers, setMarkers] = useState<LatLngExpression[]>([])
+
+  const mapRef = useRef(null)
+
+  function moveMarker() {
+    const { lat, lng } = markerPosition
+    setMarkerPosition({
+      lat: lat + 0.0001,
+      lng: lng + 0.0001,
+    })
+  }
+
+  const addMarker = (latlng: LatLngExpression) => {
+    setMarkers((c: LatLngExpression[]) => [...c, latlng])
+  }
+
+  useEffect(() => {
+    if (marker) {
+      marker.setLatLng(markerPosition)
+    }
+  }, [marker, markerPosition])
+
+  return (
+    <Map
+      ref={mapRef}
+      events={{
+        click: async e => {
+          addMarker(e.latlng)
+        },
+      }}
+      options={constants.DEFAULT_AMSTERDAM_MAPS_OPTIONS}
+      style={{
+        width: '100%',
+        height: '100vh',
+      }}
+    >
+      <ViewerContainer
```

Comment:
Same goes for this component; can it be a styled component?

Reply:
This already is...

Reply:
Done :)